### PR TITLE
Add token for light and dark scrims

### PR DESCRIPTION
**Pull request by tomgenoni**
Will be used as a background for loader dots and the value of the background for modals.

### CODE REVIEW

File: src/tokens.js [hunk 1]
```diff
@@ -2089,6 +2089,56 @@ module.exports = [
...
+                    scss: {
+                        name: '$tp-scrim__light-80',
+                        value: 'rgba(255, 255, 255, 0.8)',
+                    },
+                    javascript: {
+                        name: 'tpScrimLight80',
+                        value: 'rgba(255, 255, 255, 0.8)',
+                    },
+                    ios: {
+                        name: 'light-80',
```
Comment (danoc) [inline src/tokens.js]: The `-` isn't valid in Swift. I think this should be `light80`.

File: src/tokens.js [hunk 2]
```diff
@@ -2089,6 +2089,56 @@ module.exports = [
...
+                    scss: {
+                        name: '$tp-scrim__dark-80',
+                        value: 'rgba(0, 0, 0, 0.8)',
+                    },
+                    javascript: {
+                        name: 'tpScrimDark80',
+                        value: 'rgba(0, 0, 0, 0.8)',
+                    },
+                    ios: {
+                        name: 'dark-80',
```
Comment (danoc) [inline src/tokens.js]: The `-` isn't valid in Swift. I think this should be `dark80`.

File: src/tokens.js (hunk 1 above)
Comment (tomgenoni) [reply src/tokens.js]: Done

**Comment by tomgenoni** [comment]
@danoc This is ready for another review.

**Pull request merged**